src/components/kubectl/port-forward.ts

```diff
@@ -62,7 +65,8 @@ export async function portForwardKubernetes (explorerNode?: any): Promise<void>
             // The pod is described by the open document. Skip asking which pod to use and go straight to port-forward.
             const podSelection = portForwardablePods.pod;
             const portMapping = await promptForPort(podSelection);
-            portForwardToPod(podSelection, portMapping);
+            const namespace = portForwardablePods.namespace;
```

Comment:
since this var is only used once, we can probably remove it, and replace the call below with the direct reference to `portForwardablePods.namespace`